build.js

```diff
@@ -37,7 +37,7 @@ const pkgOptions = {
 
 // context options for qunit tests in qunit/
 const qunitOptions = {
-    ...!production ? { sourcemap: "linked" } : {},
+    sourcemap: "linked",
```

Comment:
I verified that we don't ship qunit bundles in our debs/rpms or the .tar.xz, so the big map files don't impact them. :+1: